blocks/library/columns/index.js

```diff
@@ -54,35 +52,21 @@ export const settings = {
 			type: 'number',
 			default: 2,
 		},
-		align: {
-			type: 'string',
-		},
 	},
 
-	description: __( 'A multi-column layout of content.' ),
-
-	getEditWrapperProps( attributes ) {
-		const { align } = attributes;
-
-		return { 'data-align': align };
+	supports: {
+		align: [ 'wide', 'full' ],
 	},
 
-	edit( { attributes, setAttributes, className, focus } ) {
-		const { align, columns } = attributes;
+	description: __( 'A multi-column layout of content.' ),
+
+	edit( { attributes, setAttributes, className, isSelected } ) {
+		const { columns } = attributes;
 		const classes = classnames( className, `has-${ columns }-columns` );
 
 		return [
-			...focus ? [
-				<BlockControls key="controls">
-					<BlockAlignmentToolbar
-						controls={ [ 'wide', 'full' ] }
-						value={ align }
-						onChange={ ( nextAlign ) => {
-							setAttributes( { align: nextAlign } );
-						} }
-					/>
-				</BlockControls>,
-				<InspectorControls key="inspector">
+			isSelected && (
+				<InspectorControls>
```

Comment:
I might have made the opposite claim previously, but since this is now part of the top-level return array, won't this need a `key`?

Reply:
Yes, it needs :)